registry/lombard/calldata-lbtc-redeem-mainnet.json

```
@@ -0,0 +1,74 @@
+{
+  "$schema": "../../specs/erc7730-v1.schema.json",
+  "context": {
+    "$id": "LBTC",
+    "contract": {
+      "deployments": [
+        {
+          "chainId": 1,
+          "address": "0x8236a87084f8B84306f72007F36F2618A5634494"
+        }
+      ],
+      "abi": [
+        {
+          "type": "function",
+          "name": "redeem",
+          "inputs": [
+            {
+              "name": "scriptPubkey",
+              "type": "bytes",
+              "internalType": "bytes",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            },
+            {
+              "name": "amount",
+              "type": "uint256",
+              "internalType": "uint256",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        }
+      ]
+    }
+  },
+  "metadata": {
+    "owner": "Lombard",
+    "info": {
+      "legalName": "Lombard Finance",
+      "url": "https://www.lombard.finance/"
+    }
+  },
+  "display": {
+    "formats": {
+      "redeem(bytes,uint256)": {
+        "intent": "Redeem LBTC",
+        "fields": [
+          {
+            "label": "Script Pubkey",
+            "format": "raw",
+            "path": "#.scriptPubkey.[]"
```

```suggestion
            "path": "#.scriptPubkey"
```
To remove the warning from the linter